### PR TITLE
HSC-24: Added chief complaints coded

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

**Comment by reagan-meant** [comment]
Need french locale proof reading c.c @rbuisson

File: configuration/concepts/history_and_examination.csv [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
...cted labor;Nocturia;Nausea;Nausea and vomiting;Mental status changes;Memory loss;Swelling;Breast lump;Localized superficial swelling/mass;Joint swelling;Irregular menstrual cycle;Injury;Infertility;Hoarseness;Hiccups;Hemoptysis;Hemiplegia and hemiparesis;Blood in vomit;Heartburn;Hearing loss;Headache;Gingival and periodontal diseases;Gas/bloating;Fracture;Abdominal pain;Tinnitus;Agitation;Altered consciousness;Amenorrhea;Aphasia;Arthralgia;Asthenia;Checkup;Voiding burns;Delirium;Watery diarrhea;Bloody diarrhea;Pain;Epigastric pain;Hypogastric pain;Dyspepsia;Dyspnea;Dysuria;Discharge from the eyes;Runny nose;Rashes;False needs;Chills;Halitosis;Hematuria;Hematemesis;Innapetence;Irritability;Lethargy;Leucorrhée;Lipothymie;Discomfort;Mass;Myalgia;Myasthenia;Nasal obstruction;Odinophagia;Otalgia;Otorrhea;Loss of consciousness;Weight loss;Wound;Pollakiuria;Polydipsia;Polyphagia;Hypertensive surge;Weight gain;Dermatological problems;Anal pruritus;Dermatological pruritus;Vaginal itching;Pyrosis;Rectorragie;Regurgitation;Delayed periods;Profuse sweats;Syncope;Tenesmus;Cough;Visual disturbance;Dizziness;Assault by animal suspected of rabies;Abnormal phenomenon;Breathing difficulties,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/history_and_examination.csv]: Let's rename the French to 'Motif de consultation'

File: configuration/concepts/history_and_examination.csv [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+b1c1f2f2-fe92-4aae-b5b0-df1db4fdd2de,,CIEL:160531,Non-Coded Chief Complaint,Plainte du chef non codée,Non-Coded Chief Complaint,Plainte du chef non codée,,,Misc,Text,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/history_and_examination.csv]: 'Plainte du chef non codée' to be replaced by 'Motif de consultation non-codé'

File: configuration/concepts/history_and_examination.csv [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+c3949eb6-3f10-11e4-adec-0800271c1b75,,,Chief Complaint Data,Problems,,,,,Concept Details,N/A,,,Chief Complaint; Non-Coded Chief Complaint,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/history_and_examination.csv]: In French: 'Motif de consultation data'

File: configuration/concepts/history_and_examination.csv [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+4fbf4b32-4411-436c-9c3b-1a907685a0a0,,,Image,Image,,,,,Image,Complex,ImageUrlHandler,,,,,,,,,,
+c3ae29e3-9836-4df9-bcef-8d01b776854d,,,Consultation Images,Consultation Images,,,,,Misc,N/A,,,4fbf4b32-4411-436c-9c3b-1a907685a0a0,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/history_and_examination.csv]: 'Images de consultation'

File: configuration/concepts/history_and_examination.csv [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+c3998f6b-3f10-11e4-adec-0800271c1b75,,CIEL:160029,History Notes,History Notes,,,,,Misc,Text,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/history_and_examination.csv]: 'History Notes' -> 'Notes de l'historique'

File: configuration/concepts/history_and_examination.csv [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+c2a43174-c9db-4e54-8516-17372c83537f,,,Smoking History,Smoking History,,,,,Misc,Boolean,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/history_and_examination.csv]: 'Fumeur/ancien fumeur'

**Review by rbuisson — CHANGES_REQUESTED** [review]
- Separate the Chief complaints in their own file **chief_complaints.csv**
- Remove the other concepts (not related to Chief Complaint)

File: configuration/concepts/history_and_examination.csv (hunk 1 above)
Comment (reagan-meant) [reply configuration/concepts/history_and_examination.csv]: Done

File: configuration/concepts/history_and_examination.csv (hunk 2 above)
Comment (reagan-meant) [reply configuration/concepts/history_and_examination.csv]: Done

File: configuration/concepts/history_and_examination.csv (hunk 3 above)
Comment (reagan-meant) [reply configuration/concepts/history_and_examination.csv]: Done

File: configuration/concepts/history_and_examination.csv (hunk 4 above)
Comment (reagan-meant) [reply configuration/concepts/history_and_examination.csv]: Done

File: configuration/concepts/history_and_examination.csv (hunk 5 above)
Comment (reagan-meant) [reply configuration/concepts/history_and_examination.csv]: Done

File: configuration/concepts/history_and_examination.csv (hunk 6 above)
Comment (reagan-meant) [reply configuration/concepts/history_and_examination.csv]: Done

**Pull request merged**